gr-fec/lib/reed-solomon/init_rs.c

```diff
@@ -60,6 +60,9 @@ void* INIT_RS(unsigned int symsize,
         return NULL; /* Can't have more roots than symbol values! */
 
     rs = (struct rs*)calloc(1, sizeof(struct rs));
+    if (rs == NULL) {
```

Comment:
Please remove the "{" and "}" here to keep the code looking like the prior ones that return NULL.